File: Course_01/C Language/Matrizes/acima_diagonal.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int n;
        printf("Qual a ordem da matriz? ");
        scanf("%d", &n);

    int mat[n][n], soma = 0;

    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            printf("Elemento [%d,%d]", i, j);
            scanf("%d", &mat[i][j]);
        }
    }

    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            if (i < j) {
                soma += mat[i][j];
            }
        }
    }

    printf("SOMA DOS ELEMENTOS ACIMA DA DIAGONAL PRINCIPAL = %d", soma
           );

    return 0;
}

File: Course_01/Python/Matrizes/acima_diagonal.py
m = int(input("Qual a ordem da matriz? "))
n = m

matriz = [[0 for x in range(n)] for x in range(m)]

for i in range(0, m):
    for j in range(0, n):
        matriz[i][j] = int(input(f"Elemento [{i},{j}]: "))

soma = 0
for i in range(0, m):
    for j in range(0, n):
        if j > i:
            soma += matriz[i][j]
print(f"Soma dos elementos acima da diagonal principal = {soma}")